bin/start

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -eo pipefail
+
+: ${WORKERS:=4}
```

Comment:
The `:` is a bash built-in that lets the rest of the line work but is itself a no-op, the `${blah:="something"}` syntax sets a variable if it's not already set.  There's a decent explanation on [StackExchange](http://unix.stackexchange.com/questions/25425/what-does-param-value-mean#answer-25434).

tl;dr: Sets the variable if it hasn't been set yet.